addon/components/editor-plugins/template-variable-card.js

```diff
@@ -19,6 +19,9 @@ export default class EditorPluginsTemplateVariableCardComponent extends Componen
     const config = getOwner(this).resolveRegistration('config:environment');
     this.zonalLocationCodelistUri =
       config.templateVariablePlugin.zonalLocationCodelistUri;
+    this.zonalLocationCodelistUri =
+      config.templateVariablePlugin.zonalLocationCodelistUri;
```

Comment:
This URI seems to be set twice.

Reply:
You are right, looks like been a problem with merges or something